# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/AudioSink.cpp
# Commit: 7f259ef8550f
# Full Hash: 7f259ef8550fcce1495ae80747dc5d3309bb9de4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-24 21:44:48
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - After unmuting, re-initialize the AudioSink asynchronously. r=alwu
#   
#   On Windows (in particular, but also not quick on other OS, depending on the
#   hardware being used), opening a system-level audio stream (using cubeb)
#   can be very long. It's a blocking call from the MDSM thread. The MDSM can't
# ==============================================================================

diff -r 16633fa96586 -r 7f259ef8550f dom/media/mediasink/AudioSink.cpp
--- a/dom/media/mediasink/AudioSink.cpp	Tue May 24 13:09:09 2022 +0000
+++ b/dom/media/mediasink/AudioSink.cpp	Tue May 24 13:09:09 2022 +0000
@@ -34,8 +34,7 @@
 using media::TimeUnit;
 
 AudioSink::AudioSink(AbstractThread* aThread,
-                     MediaQueue<AudioData>& aAudioQueue,
-                     const AudioInfo& aInfo,
+                     MediaQueue<AudioData>& aAudioQueue, const AudioInfo& aInfo,
                      AudioDeviceInfo* aAudioDevice)
     : mInfo(aInfo),
       mAudioDevice(aAudioDevice),